### PR TITLE
Use eigen aware version of globjects.

### PR DESCRIPTION
**Pull request by dlyr**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
A Pull Request (https://github.com/cginternals/globjects/pull/390) is currently pending for globjects, allowing to use eigen as base datatype instead of glm for setUniform.
This PR is to test the behavior in Radium.

* **What is the current behavior?** (You can also link to an open issue here)
ShaderProgram convert every vector and matrix to glm types to call globject's setUniform. 

* **What is the new behavior (if this is a feature change)?**
Use the new functionality of globjects that provides eigen setUniform.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
nope.

* **Other information**:
To be intensively tested, it should not change your client code.
If anyone sends array of vector3 and vector4, I would like to have feedback (and do not have time to test it yet...)

### CODE REVIEW

**Comment by hoshiryu** [comment]
Why do you keep `src/Core/Math/GlmAdapters.hpp` then ?

**Comment by dlyr** [comment]
Well, it could be removed, but there still have some functions that do take glm only in globjects.
And the adapters is not something bad, is it ?

[Todo list for globjects](https://gist.github.com/dlyr/03b587cb31db0027c85a41c93cdf4997)

File: .gitmodules [hunk 1]
```diff
@@ -6,7 +6,8 @@
 	url = https://github.com/cginternals/glbinding.git
 [submodule "3rdPartyLibraries/OpenGL/globjects"]
 	path = 3rdPartyLibraries/OpenGL/globjects
-	url = https://github.com/cginternals/globjects.git
+	url = git@github.com:dlyr/globjects.git
```
Comment (nmellado) [inline .gitmodules]:
```suggestion
	url = https://github.com/dlyr/globjects.git
```

**Comment by dlyr** [comment]
This PR need to be tested, so I can confirm to globjects that everything is working on our side.

**Comment by nmellado** [comment]
Added link to https://github.com/cginternals/globjects/pull/390 in PR description.

**Comment by dlyr** [comment]
Closed bu #454

**Pull request closed without merging**